pyproject.toml

```
@@ -3,11 +3,108 @@
 # NOTE: at all times.
 requires = [
     # Essentials
-    "setuptools>=34.4",
+    "setuptools>=61.2",
```

While on it, let's normalize this too:
```suggestion
    "setuptools >= 61.2",
```